oak/proto/policy.proto

```diff
@@ -47,3 +48,11 @@ message GrpcTag {
   // https://github.com/google/asylo/blob/3158887cb768112516424b3e65046f3946eb4465/asylo/identity/identity.proto#L40.
   bytes authorization_bearer_token = 1;
 }
+
+// Policies related to modules, referring to the native WebAssembly node within the TCB.
+message ModuleTag {
```

Comment:
How about `WebAssemblyModuleTag`, with a field `web_assembly_module_sha256` or similar?

Reply:
Renamed.  It seems like we might want the attestation to be a serialized proto for extensibility?